### DISCUSSION TITLE
[ Section 2 - 3 Module s] Instalar versão diferente do Terraform

### DISCUSSION THREAD

**Discussion by caiojra** [chgasparoto/curso-aws-com-terraform#52] [Q&A]
Qual versão devo utilizar para fazer o curso a versão utilizada "0.14" ou anova versão "1.0"?

**Comment by chgasparoto** [comment]
@caiojra você pode usar a versão mais nova, porém se encontrar algum erro, será provavelmente devido à nova versão.

**Reply by caiojra** [reply]
Ok, obrigado.
